packages/yourloops/test/integration/assert/patient-list.assert.ts

```diff
@@ -755,10 +757,58 @@ const checkDefaultColumnsDisplay = () => {
   expect(screen.getByRole('columnheader', { name: 'Monitoring alerts' })).toBeVisible()
   expect(screen.queryByRole('columnheader', { name: 'System' })).not.toBeInTheDocument()
   expect(screen.getByRole('columnheader', { name: 'Messages' })).toBeVisible()
-  expect(screen.getByRole('columnheader', { name: 'Time In Range' })).toBeVisible()
-  expect(screen.queryByRole('columnheader', { name: 'GMI (estimated HbA1c of last 14 days)' })).not.toBeInTheDocument()
+  expect(screen.getByRole('columnheader', { name: 'TIR' })).toBeVisible()
+  expect(screen.queryByRole('columnheader', { name: 'GMI' })).not.toBeInTheDocument()
   expect(screen.getByRole('columnheader', { name: 'Below range' })).toBeVisible()
-  expect(screen.queryByRole('columnheader', { name: 'Coefficient of Variation' })).not.toBeInTheDocument()
+  expect(screen.queryByRole('columnheader', { name: 'CV' })).not.toBeInTheDocument()
   expect(screen.getByRole('columnheader', { name: 'Last data update' })).toBeVisible()
   expect(screen.getByRole('columnheader', { name: 'Actions' })).toBeVisible()
 }
+
+const checkTooltipsColumnHeader = async (dataGridRows) => {
+  const tooltipTextMonitoringAlerts = 'Hover over the icons to learn more'
+  const tooltipTextMessages = 'Messages'
+  const tooltipTextBelowRange = 'Below Range'
+  const tooltipTextManagementGlucose = 'GMI (estimated HbA1c)'
+  const tooltipTextTimeInRange = 'Time in range'
+  const tooltipTextVariant = 'Coefficient of Variation'
+  const tooltipTextLastDataUpdate = 'Last data Update'
+
+  const monitoringAlertsColumnHeader = within(dataGridRows).getByText('Monitoring alerts')
+  const messagesColumnHeader = within(dataGridRows).getByText('Messages')
+  const TimeInRangeColumnHeader = within(dataGridRows).getByText('TIR')
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Variable names are always written in `camelCase`, not `PascalCase`. Please fix all variables in the method